camera/MultiCameraApplication/java/com/intel/multicamera/MultiViewActivity.java

```diff
@@ -153,7 +162,15 @@ public void onReceive(Context context, Intent intent) {
                         }
 
                     } else if (UsbManager.ACTION_USB_DEVICE_DETACHED.equals(action)) {
-                        System.out.println(TAG + "BroadcastReceiver USB Disconnected");
+                        UsbDevice usbDevice = intent.getParcelableExtra(UsbManager.EXTRA_DEVICE);
+                        if(usbDevice.getDeviceClass() == ic_camera.getUsb_dev_id()) {
+                            Log.d(TAG, "USB camera removed detected ");
+                        }
+                        else {
+                            Log.d(TAG, "Non USB camera device disconnected ");
```

Comment:
Change Non to No

registering of receiver inside broadcast receiver is not good. May I know why we are doing inside broadcast receiver and also outside broadcast receiver?